### DISCUSSION TITLE
Can this help me speed up my linter workflow?

### DISCUSSION THREAD

**Discussion by cofuente** [actions/cache#417] [General]
Hello!

I'm wondering if adding this cache action that works so well with my main testing script caching node modules could also help me accelerate the very, very slow workflow I have with my [github/super-linter@v3](https://github.com/marketplace/actions/super-linter) linter? I have already added it as an additional step to my existing linter workflow but it only _added_ time, didn't really help make it any faster. I suspect it's because I just copy pasted the node-modules caching step when what really lags behind in that workflow is pulling the action image for the Super-Linter action itself (takes a whole 1m 36s while every other step is under 10s).

Does anyone know if this caching action can help with that particular step in Super-Linter?

**Comment by KTibow** [comment]
I don't think so. You would have to manually load it from docker. Also possibly a duplicate of #395.

**Comment by KTibow** [comment]
And it's called super-linter because it runs lots of linters, not does it quickly. You're supposed to replace, not add it to your workflow.
